Packages/io.chainsafe.web3-unity.ramp/Runtime/Scripts/RampExchangerUniversal.cs

```diff
@@ -9,20 +10,28 @@ public class RampExchangerUniversal : IRampExchanger, ILifecycleParticipant
     {
         private readonly IRampExchangerConfig config;
         private readonly ISigner signer;
-
+        private readonly IAnalyticsClient analyticsClient;
+        
         public event Action<OnRampPurchaseData> OnRampPurchaseCreated;
         public event Action<OffRampSaleData> OffRampSaleCreated;
 
         private IRampExchanger platformImplementation;
+        
 
-        public RampExchangerUniversal(IRampExchangerConfig config, ISigner signer)
+        public RampExchangerUniversal(IRampExchangerConfig config, ISigner signer, IAnalyticsClient analyticsClient)
         {
             this.signer = signer;
             this.config = config;
+            this.analyticsClient = analyticsClient;
         }
 
         public ValueTask WillStartAsync()
         {
+            analyticsClient.CaptureEvent(new AnalyticsEvent()
```

Comment:
Sneaked this one in, forgot we didn't have analytics in Ramp and Jay needed it 😢

Reply:
noice